src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return rentCost * days - 20;
+  }
+
+  if (days >= 7) {
+    return rentCost * days - 50;
+  }
+
+  return rentCost * days;
```

Comment:
Prefer using 'const' over 'let' for the 'rentCost' variable since its value is not being reassigned. This ensures the value remains constant throughout the function execution.

Reply:
The same here